packages/rest/src/http/decorators.ts

```diff
@@ -587,71 +585,10 @@ export function $route(context: DecoratorContext, entity: Type, path: string, pa
 
   setRoute(context, entity, {
     path,
-    isReset: false,
     shared: extractSharedValue(context, parameters),
   });
 }
 
-export function $routeReset(
```

Comment:
I removed this decorator entirely because we don't use it anywhere, it doesn't even really do anything!

Reply:
Are there no test cases for it?  I thought it was used if you had a route on an interface but wanted something in that interface to not use it but instead use the `@route` directly.  

Should we deprecate this or just rip it out?

Reply:
Nope, no test cases.  It was basically dead code that I never got around to implementing.  Over time we never really needed it.

I'm OK with deprecating it if folks think it's safer.